contracts/TiramisuSavingsClub.sol

```diff
@@ -24,19 +24,22 @@ contract TiramisuSavingsClub {
     Group[] public groups;
 
     mapping(address => uint) public memberToGroupId;
-    mapping(address => uint) private deposits;
-    mapping(address => uint) private withdrawals;
+    mapping(address => uint) deposits;
+    mapping(address => uint) withdrawals;
+    mapping(address => string) memberNames;
 
-    event Deposit(address depositor, uint groupId, uint amount);
-    event Withdrawal(address payee, uint groupId, uint amount);
+    event NewGroupEvent(address[] addresses, string[] names, uint ownerIndex);
+    event DepositEvent(uint groupId, uint amount, address sender, string senderName, uint newGroupBalance);
```

Comment:
I think it's a little redundant to add the "Event" suffix to every event name. Could we remove that?